src/page-multi-facility/FacilityRow.tsx

```diff
@@ -109,11 +140,18 @@ const FacilityRow: React.FC<Props> = ({
   const [model, updateModel] = useModel();
 
   const { rtData, setFacility } = useContext(FacilityContext);
+
   const [facility, updateFacility] = useState(initialFacility);
-  let useRt, facilityRtData;
+  let useRt,
+    facilityRtData,
+    latestRt = null;
   if (useFlag(["useRt"])) {
     useRt = true;
-    facilityRtData = rtData ? rtData[facility.id] : undefined;
+    facilityRtData = rtData ? rtData[facility.id] : null;
+
+    if (facilityRtData) {
+      latestRt = facilityRtData?.Rt[facilityRtData.Rt.length - 1].value;
```

Comment:
considering the conditional you shouldn't need `facilityRtData?` here?

Reply:
also see #273 , a helper function is forthcoming that should replace this assumption that the last value is always most recent (it should be a safe assumption for now but it's obviously brittle)

Reply:
Re: conditional, I thought that passed the typescript error, but I didnt realize that actually also works for the assignment.

Reply:
Adding a todo to update the helper when its ready.